main/src/library/Prelude.flix

```diff
@@ -17,6 +17,15 @@ pub enum Proxy[_] {
     case Proxy
 }
 
+///
+/// The uninterpretable `Env` effect.
+///
+/// The `Env` effect represents ..
```

Comment:
... actions that read from the environment (e.g. information about the shell, operating system, or hardware)